index.js

```diff
@@ -35,13 +42,13 @@ Ewma.prototype.insert = function insert(x) {
     var elapsed = now - self._stamp;
     self._stamp = now;
 
-    var w = Math.exp(-elapsed / self._tau);
+    var w = Math.pow(2, -elapsed / self._decay);
     self._ewma = w * self._ewma + (1.0 - w) * x;
 };
 
 Ewma.prototype.reset = function reset(x) {
     var self = this;
-    self._stamp = 0;
+    self._stamp = self._clock.now();
```

Comment:
Yep, it was a bug.
Note that we could define another function `reset()` which reset the stamp to 0, which will basically make the ewma equal next value (`dt` will be big, then `w` will be ~equal to `0`).